lib/utils.tsx

```diff
@@ -288,6 +288,10 @@ export function endsNextDayBeforeMorning(
   return isNextDay && isBeforeMorning;
 }
 
+export function eventTimesAreDefined(startTime: string, endTime: string) {
+  return startTime != undefined && endTime != undefined;
```

Comment:
```suggestion
  return startTime !== undefined && endTime !== undefined;
```

note: almost always in JS you want to use strict comparisons like [strict inequality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_inequality) because otherwise it attempts to convert types to compare them, which produces unexpected results like `"1" != 1` is false. generally you want to check if things are the same type and the same value, rather than implicitly converting and comparing the values